src/encoded/static/components/publication.js

```diff
@@ -26,7 +73,11 @@ const PublicationComponent = (props, reactContext) => {
     ];
 
     const crumbsReleased = (context.status === 'released');
-
+    const accessions = !context.datasets ? '' : context.datasets.reduce((dataset1, dataset2) => `${dataset1}&accession=${dataset2.accession}`, '');
+    const fields = ['accession', 'target', '@type', 'target', 'biosample_summary', 'lab', 'assay_term_name', 'assay', 'description', 'status'].reduce((field1, field2) => `${field1}&field=${field2}`, '');
+    const datasetsUrl = context.datasets && context.datasets.length ?
+        `/search/?${accessions}${fields}&format=json` :
```

Comment:
You can change this if you want; `FetchedData` specifies JSON, so `format=json` in the URL here is redundant, but it also doesn’t hurt anything.